subprojects/docs/src/snippets/java/multiproject/groovy/shared/build.gradle

```diff
@@ -0,0 +1,3 @@
+plugins {
+    id 'java'
```

Comment:
Not super important here as it is not shown in the doc page, but:

The recommendation would be that the "java" plugin should not be applied in isolation. So you might apply it in a convention plugin. But in a build script, you would only apply other plugins that use "java" under the hood. I would use `java-library` here.